webcompat/templates/home-page/how-it-works.html

```diff
@@ -0,0 +1,40 @@
+<section class="grid">
+  <article class="grid-row">
+    <div class="grid-cell x2">
+      <h2 class="headline-1 highlight">What is this project about?</h2>
+      <p>
+        People expect websites to work regardless of the device or browser they are using. If they are behaving differently, that is most likey a web compatibility bug we are looking for.
+      </p>
+    </div>
+<div class="grid-cell x1">
+        <h3 class="headline-4">Join Slack!</h3>
+        <p>Come and join our web compat community.</p>
+        <a href="./contact" class="button button-primary">Let's chat</a>
+      </div>
+  </article>
+</section>
+
+<section class="grid">
+  <article class="grid-row">
+    <div class="grid-cell x1">
+      <h3 class="headline-4">Meet the team</h2>
+      <p>
+        Who are the people behind the web compat project? <br /><br/>
+
+        Find out and <a href="./about" title="meet the wompats">meet the Wompats</a>.
+      </p>
+    </div>
+    <aside class="grid-cell x2">
+      <h2 class="headline-1 highlight">What is a web compat bug?</h2>
+      <p>
+        How do I know the problem I found is a webcompat bug and not something else? Simple! <br />
+      <br />
+        1. Open the site in another browser! The problem doesn’t exist? <br />
```

Comment:
Can we use an ordered list here? Then we can ditch the `<br />` and just use CSS.

Reply:
That actually makes so much sense.